draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -221,70 +216,69 @@
+          If that statement is true, then the AS MUST register an AS0 ASPA.
+        </t>
+        <t>
+          Normally, the Provider ASes of a CAS would be congruent for the address family combinations {AFI 1 (IPv4), SAFI 1} and {AFI 2 (IPv6), SAFI 1}.
+          Exceptions to this are expected to be rare.
+          In any case, the CAS MUST list the union of all Provider ASes applicable to the address family combinations stated above in the SPAS and MUST also include any non-transparent RS AS(es) at which it is an RS-client.
+          In the procedures for the AS path verification described in this document (<xref target="pair-validation"/>, <xref target="verif"/>), the SPAS is always considered to be uniformly applicable to {AFI 1 (IPv4), SAFI 1} and {AFI 2 (IPv6), SAFI 1}.
+        </t>
+        <t>
+          A compliant AS or Route Server AS (RS AS) MUST have an ASPA.
+          An RS AS SHOULD register an AS0 ASPA.
+          An AS SHOULD NOT have more than one ASPA.
+          Such a practice helps prevent race conditions during ASPA updates that might affect prefix propagation.
+          The software that provides hosting for ASPA records SHOULD support enforcement of this practice.
+          During a transition process between different certificate authority (CA) registries, the ASPA records SHOULD be kept identical in all relevant registries.
+        </t>
+        <t>
+          Normally, a VAP (see <xref target="rec1"/>) is not expected to contain both an AS 0 and other Provider ASes, but an unexpected presence of AS 0 has no influence on the AS path verification procedures (see <xref target="pair-validation"/>, <xref target="verif"/>).
+        </t>
+        <t>
+          Each of the two ASes in a mutual-transit pair MUST register its ASPA including the other AS in its SPAS.
+        </t>
+        <t>
+          The ASes on the boundary of an AS Confederation MUST register ASPAs using the Confederation's global ASN as the CAS.
+        </t>
+      </section>
+    <section title="Hop-Check Function" anchor="pair-validation">
       <t>
-        Let AS x and AS y represent two unique ASes.
-        A provider authorization function, authorized(AS x, AS y), checks if the ordered pair of ASNs, (AS x, AS y), has the property that AS y is an attested provider of AS x per VAP-SPAS of AS x.
-        The VAP-SPAS table is assumed to be organized in such a way that it can be queried to check (1) if a specified CAS = AS x has an entry (i.e., SPAS listed), or (2) if for a given (AS x, AS y) tuple, AS x is listed in the VAP-SPAS as a provider associated with CAS = AS y.
-        By term "Provider+" the function should signal if AS y plays role of Provider, non-transparent RS, or mutual-transit neighbor.
-        This function is specified as follows:
+          Let AS x and AS y represent two unique ASes.
```

Comment:
We have discussed it during previous review. It is not a hop based, it just applied to an ordred pair.